docs/pages/index.mdx

```diff
@@ -2,39 +2,134 @@ import ImageRow from "@/components/ImageRow";
+
+This process serves two purposes -- first, whenever the Origin re-advertises with the federation's Director, the Director can verify the authenticity of those advertisements through public/private key asymmetric cryptography by looking at the Registry's stored public key for that Origin and namespace. Second, the Registry's persistent database prevents other Origins from registering namespaces under an already-registered namespace without first proving they're allowed to do so by the namespace owner (i.e. the entity that possesses the appropriate private key).
+
+## Making Bytes Accessible and Moving Them -- A First Look Under The Hood
+
+This section provides a simplified example of how data is made accessible and moved within the OSDF. In particular, it elides the OSDF’s Staging/Caching infrastructure and any discussion of authorization tokens.
+
+Pelican serves two sides of the same coin -- Data owners who want to federate their data from wherever it lives natively, and data consumers who want to access and compute on data wherever they need it.
+
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-repo-and-consumer.png"}>
+	The federation's core goal is connecting data owners and data consumers.
+</ImageRow>
+
+
+As such, the primary prerequisite for data to be moved via a Pelican federation is for a data owner to make their data accessible to the federation. This happens when an Origin is placed in front of the repository and registered with the federation. While federations like the OSDF *may* wish to control or filter any Origin registrations to vet the data they make available, this example assumes the Origin's registration is automatically approved. The red arrow in the following graphic represents the vetting/approval step, should the federation require it.
+
+<ImageRow alt={"Pelican and OSDF"} src={"/pelican/arch-origin-registration.png"}>
+	When the Origin starts, it's told which federation it belongs to. It discovers the
+	hostnames for its Registry and Directory by using the federation root to construct
+	the URL "https://osg-htc.org/.well-known/pelican-configuration", which contains a JSON
+	detailing the federation's central services.
+
+	Next, the Origin registers its namespace and public key with the Registry, proving that it
+	owns the corresponding private key. Finally, the Origin begins advertising its namespace
+	information and hostname to the Director.
+</ImageRow>
+
+While somewhat simplified, this example illustrates the process origins must take to make themselves known within the federation. After completing these steps, the objects from the Data Repository are available via Pelican.
+
+The next step is for the data consumer to actually *move* the data. Pelican assumes the data consumer already knows the federation that provides the data they want, along with the name of the object within the federation. These two pieces of information are combined and provided to the Client as a `pelican://`-schemed URL
```

Comment:
With our ongoing work for client list functionality, would we expect this to be changed soon since we do (and want to?) provide a way for user to discover data?

Reply:
I'd argue yes and know -- providing a way to `ls` for data is still pretty far from providing a way to discover data, because all `ls` tells you is the name of some objects, and nothing about what information they contain, how the user came to be looking for data at a given path prefix, etc. In order to say "I want to compute on weather data from the `/ncar`  namespace," the user still has to be told that they should look there for weather data, even if they can now discover the names of the objects it hosts.